### PR TITLE
Feat - added cart page

### PR DESCRIPTION
**Pull request by Mohasin-Haque**
* Added Cart Page 

See the preview here :
image

### CODE REVIEW

File: pages/filter.html [hunk 1]
```diff
@@ -27,8 +27,8 @@
         <div class="navbar-links-ecom">
             <ul>
                 <li><a href="../index.html">Home</a></li>
-                <li><a href="#">cart</a></li>
-                <li><a href="#">wishlist</a></li>
+                <li><a href="./wishlist.html"><i class="bi bi-heart"></i>Wishlist</a></li>
+                <li><a href=".cart.html"><i class="bi bi-cart"></i>Cart</a></li>
```
Comment (LordUttam) [inline pages/filter.html]: slash missing ./cart.html

File: pages/login.html [hunk 2]
```diff
@@ -27,8 +27,8 @@
         <div class="navbar-links-ecom">
             <ul>
                 <li><a href="../index.html">Home</a></li>
-                <li><a href="#">cart</a></li>
-                <li><a href="#">wishlist</a></li>
+                <li><a href=".wishlist.html"><i class="bi bi-heart"></i>Wishlist</a></li>
```
Comment (LordUttam) [inline pages/login.html]: slash missing ./wishlist.html

File: pages/login.html [hunk 3]
```diff
@@ -27,8 +27,8 @@
...
+                <li><a href=".cart.html"><i class="bi bi-cart"></i>Cart</a></li>
```
Comment (LordUttam) [inline pages/login.html]: slash missing ./cart.html

File: pages/menswear.html [hunk 4]
```diff
@@ -28,8 +28,8 @@
         <div class="navbar-links-ecom">
             <ul>
                 <li><a href="./index.html">Home</a></li>
-                <li><a href="#">cart</a></li>
-                <li><a href="#">wishlist</a></li>
+                <li><a href=".wishlist.html"><i class="bi bi-heart"></i>Wishlist</a></li>
```
Comment (LordUttam) [inline pages/menswear.html]: same problem in menswear and signup pages

File: pages/cart.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>navbar | fastcart</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="author" content="Mohasin Haque">
+    <meta name="description" content="Responsive navbar for e commerce sites.">
```
Comment (xii-vi) [inline pages/cart.html]: Change description here

File: pages/cart.html [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+    <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+</head>
+
+<body>
+    <nav class="navbar-ecom">
+        <a href="../index.html" class="brand-title-ecom">fastcart</a>
+        <a href="#" class="toggle-button-ecom">
+            <span class="bar-ecom"></span>
+            <span class="bar-ecom"></span>
+            <span class="bar-ecom"></span>
```
Comment (xii-vi) [inline pages/cart.html:21]: Why an empty span?

File: pages/product.html [hunk 7]
```diff
@@ -126,9 +126,7 @@
                     <div class="card-content">
                         <h2 class="card-heading">Black Zari Banarasi Saree with Blouse
                         </h2>
-                        <p class="card-author">Rs. 3,390 only</p
-                    </div>
-                    <div class="card-actions">
+                        <p class="card-author">Rs. 3,390 only</p </div> <div class="card-actions">
```
Comment (LordUttam) [inline pages/product.html]: &gt; missing in closing &lt;/p&gt; tag.  Also reformat div.card-actions to next line.

File: styles/cart.css [hunk 8]
```diff
@@ -0,0 +1,137 @@
+.cards-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (xii-vi) [inline styles/cart.css]: Make utility class out of these three

**Review by xii-vi — APPROVED** [review]
Added few comments. Kindly go through them before merging PR.

**Review by LordUttam — COMMENTED** [review]
Some problems in links and formatting. Rest looks good to me.

File: pages/cart.html (hunk 6 above)
Comment (LordUttam) [reply pages/cart.html:21]: I think that's for hamburger icon. It will be better to use a readymade icon instead.

**Pull request merged**